include/drivers/console.h

```diff
@@ -53,7 +53,7 @@ typedef struct console {
  * registered that way can be unregistered/reconfigured with below functions.
  */
 /* Remove a single console_t instance from the console list. */
-int console_unregister(console_t *console);
+console_t *console_unregister(console_t *console);
```

Comment:
Looks like this was wrong before. Would be good to call this out in commit message.

Reply:
Will put in a separate commit for clarity.